pcsx2/Patch.h

```diff
@@ -128,3 +131,8 @@ extern void PatchesVerboseReset();
 // regardless, they don't seem to have an implementation anywhere.
 // extern int  AddPatch(int Mode, int Place, int Address, int Size, u64 data);
 // extern void ResetPatch(void);
+
+//Swaps endianess of InputNum
+//ex. 01020304 -> 04030201
+//Could be made more portable if we remove the patch_data_type bar
+extern u64 SwapEndian(u64 InputNum, patch_data_type);
```

Comment:
Add a space after `//` for consistency with other comments

Do you also perhaps mean `var` instead of `bar` in your comment?

Reply:
Fixed in https://github.com/PCSX2/pcsx2/pull/4255/commits/006e946060582292f3e9b935f97268407d796010